installer/common/shared.sh

```diff
@@ -45,7 +45,7 @@ export KAFKA="${KAFKA:-CloudflowManaged}"
 
 # Flink
 export flinkReleaseName="cloudflow-flink"
-export flinkOperatorChartVersion="0.8.1"
+export flinkOperatorChartVersion="0.8.2"
```

Comment:
We are updating this in the Enterprise installer also, right?

Reply:
yes